cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java

```diff
@@ -352,7 +352,7 @@ private MetricQueryResult executeQuery(MetricQueryRequest queryRequest) throws E
       endTime = TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis());
     }
 
-    return decorate(queryResult, timeRange.getStart(), endTime);
+    return decorate(queryResult, timeRange.getStart(), endTime, timeRange.getResolutionInSeconds().intValue());
```

Comment:
just do, timeRange.gerResolutionInSeconds(), you need to add  the unit - "s" as we talked before.